### DISCUSSION TITLE
A list of potentially useful collaborative spaces.

### DISCUSSION THREAD

**Discussion by daank-c** [veryengineer/collab#2] [Show and tell]
Hello everyone,

I wanted to start a "show and tell" discussion to highlight some collaborative spaces that I've found useful in the past and that might be beneficial for very.engineer projects in the future. I would also love to hear about any other spaces that you all have found to be effective for collaboration.

**Comment by daank-c** [comment]
Here are some examples of collaboration spaces that **I** have personally found useful:  (i'm big on whiteboards xD)

### Figma  (free)

Figma is a web-based, collaborative interface design tool that allows multiple people to work on a design project at the same time. It also allows for real-time collaboration and version control, making it a great tool for teams working on user interface design, wireframes, and mockups. It also allows for the integration with other tools such as Gitbook, Trello and Notion. See more integrations [here](https://www.figma.com/product-integrations/).

This is an example of a Figma Jam environment:

![Screenshot 2023-01-29 6 16 12 PM](https://user-images.githubusercontent.com/102710924/215358719-1f21ab99-f513-4a65-bf8a-77afcd978032.png)


### Miro (free)
 
Miro is a digital whiteboard platform that enables teams to visually collaborate, brainstorm, plan, and organize ideas in real-time. It's an online platform that allows for an infinite canvas where you can add sticky notes, drawings, diagrams, and more. Miro allows for real-time collaboration and version control, making it easy to keep track of changes and revisions. It also allows for the integration with other tools such as Trello and Asana. See more integrations [here](https://miro.com/integrations/).

This is an example of a Miro environment:

![Screenshot 2023-01-29 6 23 23 PM](https://user-images.githubusercontent.com/102710924/215359044-8faeea43-6a46-4323-8fbc-0ef7eeaa1fb2.png)

**Comment by tosufever** [comment]
is there a need for something like a discord?

**Reply by patricklodder** [reply]
Feel free to set one up.
